src/ex6/client/src/components/todo-element.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import { deleteItem, markItemAsOld, changeItemStatus } from "../item_client.js";
+import PropTypes from 'prop-types';
+
+export function TodoElement(props) {
+  const [todoClassName, setTodoClassName] = useState("todo-li existing-todo")
+
+  const todo = props.todo;
+  const doneTime = todo.done === null? "" : `Done at ${todo.done.slice(11,16)} ${todo.done.slice(0,10)}`;
+  const checked = todo.status? "checked" : "";
+
+  useEffect(() => {
+    if (todo.isNew) {
+      setTodoClassName("todo-li existing-todo animation-add-todo");
+      markItemAsOld(todo);
+    }
+  },[todo])
+
+  const animationEndHandler = async () => {
+    if (todoClassName === "todo-li existing-todo animation-add-todo") {
+      setTodoClassName("todo-li existing-todo")
+    } else {
+      await props.updateTodos();
```

Comment:
let's make this more readable and descriptive

Reply:
done